CSIP7: Create testCase.xml

koit opened the pull request:
#91

koit committed:
Create testCase.xml for CSIP7

PhillipTommerholt commented on corpora/csip/metadata/metshdr/CSIP7/testCase.xml:
```diff
@@ -1,23 +1,51 @@
+            date" is tested for using XML Schema validation as mets.xsd defines
+            the attribute as type="xsd:dateTime". -->
+      
+    <!--    Requirement "mets/metsHdr/@CREATEDATE value MUST be the actual date
```
I think it would be possible and actually make sense to test if the date is after today.

koit replied:
Thanks, good point! Fixed now.

carlwilson replied:
Beware here be dragons, time zones or even server clock discrepancies, and other oddities might make this an annoying test IRL. Perhaps with a suitable buffer-zone?

koit replied:
Agreed, less than a day before the cliff is way too close. I changed it to 2037-01-01.

PhillipTommerholt commented on corpora/csip/metadata/metshdr/CSIP7/testCase.xml:
```diff
@@ -1,23 +1,51 @@
+      </error>
+      <corpusPackages>
+        <package name="mets-xml_metsHdr_CREATEDATE_not_exist" isValid="FALSE">
+          <path></path>
```
insert path

PhillipTommerholt commented on corpora/csip/metadata/metshdr/CSIP7/testCase.xml:
```diff
@@ -1,23 +1,51 @@
+          </description>
+        </package>
+        <package name="minimal_IP_with_1_representation" isValid="TRUE">
+          <path></path>
```
insert path

ltaimre committed:
added test-package mets-xml_metsHdr_CREATEDATE_not_exist

carlwilson committed:
Merge branch 'integration' into csip/metsHdr/csip7

koit committed:
Update testCase.xml & .xsd

Add rule 2 (date in the future).
Add paths.
Update testCase.xsd to the version with `package/@isImplemented`.

koit commented:
To stick to our two-stage process of testCase first, packages after, I only fixed the testCase.xml, not the IP. Let's merge this PR and I'll do the IP then.

koit committed:
Merge branch 'integration' into csip/metsHdr/csip7

koit committed:
Set the future date to a safer value of 2037-01-01

PhillipAasvangTommerholt committed:
updated testcase.xml and zipped a package

updated testcase.xml and zipped a package

aebkmd reviewed (approved):
Looks fine!
